src/tools/impl/file_system.py

```
@@ -60,7 +61,10 @@ def _render_diff_args(args: dict, config: dict) -> str:
         try:
             edits = json.loads(edits)
         except (json.JSONDecodeError, ValueError):
-            pass
+            try:
+                edits = json.loads(repair_json(edits))
+            except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)